scripts/ignite.sh

```diff
@@ -0,0 +1,5 @@
+kill -9 `lsof -i:10800|awk '{print $2}'|grep -v PID`
+cd /root/git/apache-ignite-2.8.1-bin/bin/
```

Comment:
Let us add one deployment script for Ignite container, which will be a major usage for Ignite.

Reply:
fixed in 3bfa97f